### DISCUSSION TITLE
Zigbee and time of day question.

### DISCUSSION THREAD

**Discussion by paulz-gh** [Koenkk/zigbee2mqtt#24476] [General]
Hi, 
Last week I have received a Tuya rain sensor https://www.zigbee2mqtt.io/devices/RB-SRAIN01.html#tuya-rb-srain01
And it seems to regard the moist Dutch night are as rain ;-)
But there is one other thing that surprises me. One of the measurements is Illuminance maximum for the last 24 hours.
I noticed the value resets almost exactly at midnight local time ( 2200 UTC) 
```
2024-10-24T22:00:55.575527328Z 5399
2024-10-24T22:00:55.697730407Z 5399
2024-10-24T22:00:55.822090311Z 5399
2024-10-24T22:01:55.574317808Z 36
2024-10-24T22:01:55.6991973Z   36
```

How can the sensor know the local time ? There is no way to set it.
Is it part of the zigbee packets ?

Just curious,
   Paul

**Comment by Koenkk** [comment]
Yes, a device can request the time over Zigbee

**Reply by paulz-gh** [reply]
Are devices that show the genTime cluster requesting time ?

**Reply by Koenkk** [reply]
Yes (but I'm not sure if all device with `genTime` actually request the time)
